choosing RESET_STREAM or RESET_STREAM_AT (#30)
quicwg/reliable-stream-reset: draft-ietf-quic-reliable-stream-reset.md

# Resetting Streams

When a sender wants to reset a stream but also deliver some bytes to the receiver,
the sender sends a RESET_STREAM_AT frame with the Reliable Size field specifying
the amount of data to be delivered.

When resetting a stream without the intent to deliver any data to the receiver,

> the sender MAY use a RESET_STREAM_AT frame with a Reliable Size of zero instead of
> using a RESET_STREAM frame. These two are identical and the behavior of

Author: ```suggestion
the sender uses a RESET_STREAM frame (Section 3.2 of {{!RFC9000}}). The sender
MAY also use a RESET_STREAM_AT frame with a Reliable Size of zero in place of a
a RESET_STREAM frame. These two are identical and the behavior of
```

# Resetting Streams

> When resetting a stream requesting some bytes to delivered to the peer

Author: ```suggestion
When resetting a stream requesting some bytes to be delivered to the peer
```
grammar.

> When resetting a stream while requesting some bytes to delivered to the peer

Author: ```suggestion
When resetting a stream requesting some bytes to delivered to the peer
```
"while" is ambiguous.
